lib/benchmark_driver/output/record.rb

```diff
@@ -51,7 +51,7 @@ def report(result)
   private
 
   def save_record
-    jobs = @benchmark_metrics
+    # jobs = @benchmark_metrics
```

Comment:
Looks like the line should be removed instread

Reply:
:+1: https://github.com/benchmark-driver/benchmark-driver/commit/bdee14f8b7dc8130a4dd99d80bc0db2023ce75a3